cmd/minikube/cmd/addons/addons.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var AddonsCmd = &cobra.Command{
```

Comment:
What do you think about adding this to `minikube config set/unset` instead?  And utilizing the set functions and callbacks to enable/disable the addons in whichever way you want.  Maybe even nesting the addon command within config. `minikube config addon enable dashboard`.  Maybe a bit verbose...